### PR TITLE
Loader update

### PR DESCRIPTION
**Pull request by khandelwal**
This pull request changes the data loading code to reflect the new format of the YAML files - especially around addresses. We are changing how we represent things in the YAML - most especially by moving a lot of the data munging to the YAML. 

This has the advantage that the YAML files are more consistent. It also means that this loader is simpler. 

See: https://github.com/18F/foia/pull/101

### CODE REVIEW

**Comment by khandelwal** [comment]
Fixes #405

**Comment by khandelwal** [comment]
@ramirezg Will you review this?

File: foia_hub/scripts/load_agency_contacts.py [hunk 1]
```diff
@@ -31,80 +31,66 @@ def check_urls(agency_url, row, field):
         return row_url
 
 
-TTY_RE = re.compile('\(?\d{3}\)? \d{3}-\d{4} \(TTY\)')
+TTY_RE = re.compile('\d{3}-\d{3}-\d{4} \(TTY\)')
```
Comment (ramirezg) [inline foia_hub/scripts/load_agency_contacts.py]: All numbers should be formatted properly at this point. Is there a reason for keeping the regex? Also, what if a TTY number has an extension?

File: foia_hub/scripts/load_agency_contacts.py [hunk 2]
```diff
@@ -31,80 +31,66 @@ def check_urls(agency_url, row, field):
...
 ADDY_RE = re.compile('(?P<city>.*), (?P<state>[A-Z]{2}) (?P<zip>[0-9-]+)')
```
Comment (ramirezg) [inline foia_hub/scripts/load_agency_contacts.py]: ADDY_RE is unused.

**Comment by ramirezg** [comment]
I think we'll need a migration to delete invalid addresses like [Office of Operations Coordination and Planning](https://foia.18f.us/contacts/department-of-homeland-security--office-of-operations-coordination-and-planning) either in this PR or another. Or, we could just get new addresses for the 8 that are now missing.

**Comment by khandelwal** [comment]
I'm thinking let's get the eight addresses that are missing (if we can). That seems a better long term solution.

**Pull request merged**